posthog/temporal/batch_exports/snowflake_batch_export.py

```diff
@@ -469,7 +459,7 @@ async def flush_to_snowflake(
         record_iterator = iter_records(
             client=client,
             team_id=inputs.team_id,
-            interval_start=inputs.data_interval_start,
+            interval_start=data_interval_start,
```

Comment:
This is actually a bug and means we could have produced some duplicates when exporting to Snowflake...

Reply:
Granted, it's only in the not super-common case were we first failed and had to resume.